models/models.js

```diff
@@ -31,21 +31,37 @@ const fetchAllTopics = (sort_by='slug', order='ASC', slug) => {
 }
 
 const fetchArticleById = (article_id) => {
-    if(isNaN(article_id)){
-        return Promise.reject({status: 400, msg:'Bad request'})
+    const regex = /\d\(comment_count\)/
```

Comment:
based on my comment above, you'll probably very quickly realise you don't actually need all of this code! :D